ocsci/main.py

```diff
@@ -113,4 +114,5 @@ def main(arguments):
         '-p', 'ocsci.pytest_customization.marks',
         '-p', 'ocsci.pytest_customization.ocsci_logging',
     ])
+    utils.add_path_to_env_path(os.path.expanduser(ocsci.config.RUN['bin_dir']))
```

Comment:
If we will need to do more post init action in the future we can create some function for this purpose which will contain those and call it from here.  But I am ok for this PR, just FYI if you will add some more next time.